tools/to_toit_source.toit

```diff
@@ -237,6 +237,7 @@ main args/List:
   print "  DIGICERT_HIGH_ASSURANCE_EV_ROOT_CA,"
   print "  ISRG_ROOT_X1,"
   print "  STARFIELD_CLASS_2_CA,"
+  print "  COMODO_AAA_SERVICES_ROOT,"
```

Comment:
Maybe put this next to the other COMODO certificate?

Reply:
done.